### DISCUSSION TITLE
Does kor remember removed resources?

### DISCUSSION THREAD

**Discussion by northys** [yonahd/kor#131] [Q&A] [answered]
I've deployed kor and it found some legacy stuff which I've removed but it kept exporting it to the prometheus until recent upgrade (== container restart). Is it an expected behaviour?

At first it scared me because I thought I've deleted wrong namespace because I still saw it in grafana dashboard.

Thanks for the clarification!

**Comment by yonahd** [comment, marked as answer]
I see what you mean.
It seems the Gauge is not going down meaning it stores the state and is not resetting it each iteration.
This is a bug.

Fixed here https://github.com/yonahd/kor/pull/132
